bccsp/pkcs11/pkcs11.go

```diff
@@ -800,6 +801,18 @@ func (csp *Provider) ecPoint(session pkcs11.SessionHandle, key pkcs11.ObjectHand
 	return ecpt, oid, nil
 }
 
+func (csp *Provider) handleSessionReturn(err error, session pkcs11.SessionHandle) {
+	if err != nil {
+		re := regexp.MustCompile(".*0xB.:\\sCKR.+")
```

Comment:
Perhaps compile the regexp once in a global variable?

Reply:
Fixed